tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,23 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    assert len(items) == 0
+    assert item_b not in items
```

Comment:
There is no need to check if `item_b` is in `items` because we already know `items` is empty and can't have anything in it.